types/wallet.d.ts

```diff
@@ -21,13 +22,121 @@ declare module 'wallet' {
       ): Promise<TransactionResult>;
     }
 
+    interface MultiSigWallet extends ContractBase {
+      owners(): Promise<Address[]>;
+      required(): AnyNumber;
+
+      addOwner(
+        owner: Address,
+        options?: TransactionOptions
+      ): Promise<TransactionResult>;
+
+      removeOwner(
+        owner: Address,
+        options?: TransactionOptions
+      ): Promise<TransactionResult>;
+
+      replaceOwner(
+        owner: Address,
+        newOwner: Address,
+        options?: TransactionOptions
+      ): Promise<TransactionResult>;
+
+      changeRequirement(
+        required: AnyNumber,
+        options?: TransactionOptions
+      ): Promise<TransactionResult>;
+
+      submitTransaction(
+        destination: Address,
+        value: AnyNumber,
+        data: string,
+        options?: TransactionOptions
+      ): Promise<TransactionResult>;
+
+      confirmTransaction(
+        transactionId: AnyNumber,
+        options?: TransactionOptions
+      ): Promise<TransactionResult>;
+
+      revokeConfirmation(
+        transactionId: AnyNumber,
+        options?: TransactionOptions
+      ): Promise<TransactionResult>;
+
+      executeTransaction(
+        transactionId: AnyNumber,
+        options?: TransactionOptions
+      ): Promise<TransactionResult>;
+
+      getConfirmationCount(transactionId: AnyNumber): Promise<AnyNumber>;
```

Comment:
Return value should be `Promise<BigNumber>`. Web3 takes as a parameter any of string, number, and BigNumber but returns always BigNumber.